app/build.gradle

```diff
@@ -82,67 +82,95 @@ dependencies {
+    compile 'com.android.support.constraint:constraint-layout:1.0.2'
 
-    /* Support Libraries */
+    /* Android Support Libraries
+     * https://developer.android.com/topic/libraries/support-library/index.html */
     compile "com.android.support:appcompat-v7:$support_lib_version"
     compile "com.android.support:cardview-v7:$support_lib_version"
-    compile "com.android.support:recyclerview-v7:$support_lib_version"
     compile "com.android.support:design:$support_lib_version"
     compile "com.android.support:palette-v7:$support_lib_version"
-    compile 'com.android.support.constraint:constraint-layout:1.0.2'
-    compile 'com.google.android:flexbox:0.2.5'
+    compile "com.android.support:recyclerview-v7:$support_lib_version"
 
-    /* Mosby / MVP */
-    compile 'com.hannesdorfmann.mosby:viewstate:2.0.1'
+    /* ErrorView
+     * https://github.com/xiprox/ErrorView */
+    compile 'tr.xip.errorview:library:3.0.0'
 
-    /* Retrofit and JSON */
-    compile 'com.squareup.okhttp3:okhttp:3.6.0'
-    compile 'com.squareup.okhttp3:logging-interceptor:3.6.0'
-    compile "com.squareup.retrofit2:retrofit:$retrofit_version"
-    compile 'com.squareup.retrofit2:converter-jackson:2.2.0'
-    compile 'com.jakewharton.retrofit:retrofit2-rxjava2-adapter:1.0.0'
-    compile 'com.fasterxml.jackson.core:jackson-core:2.8.6'
-    compile 'com.fasterxml.jackson.core:jackson-databind:2.8.6'
-    /* compile 'com.github.jasminb:jsonapi-converter:0.7' */
-    compile('com.github.Kevinrob:jsonapi-converter:090d7a89426c4765ea08edb655d5c0aa7e3c7d5d'){
+    /* Facebook's Fresco
+     * Image loading
+     * https://github.com/facebook/fresco */
+    compile "com.facebook.fresco:fresco:$fresco_version"
+    compile "com.facebook.fresco:animated-gif:$fresco_version"
+
+    /* Google's Flexbox
+     * https://github.com/google/flexbox-layout */
+    compile 'com.google.android:flexbox:0.3.0'
+
+    /* Jackson
+     * JSON parsing
+     * https://github.com/FasterXML/jackson-core */
+    compile "com.fasterxml.jackson.core:jackson-core:$jackson_version"
+    compile "com.fasterxml.jackson.core:jackson-databind:$jackson_version"
+
+    /* JSONAPI converter
+     * https://github.com/Kevinrob/jsonapi-converter */
+    compile('com.github.Kevinrob:jsonapi-converter:090d7a89426c4765ea08edb655d5c0aa7e3c7d5d') {
         changing=true
     }
 
-    /* Rx */
-    compile 'io.reactivex.rxjava2:rxandroid:2.0.1'
-    compile 'io.reactivex.rxjava2:rxjava:2.0.6'
-    compile 'com.jakewharton.rxbinding2:rxbinding-kotlin:2.0.0'
-
-    /* Fresco / Image Loading */
-    compile 'com.facebook.fresco:fresco:1.1.0'
-    compile 'com.facebook.fresco:animated-gif:1.1.0'
+    /* Kotlin
+     * https://kotlinlang.org/ */
+    compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+    testCompile "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+    testCompile "org.jetbrains.kotlin:kotlin-test-junit:$kotlin_version"
```

Comment:
🤔  Should we have Kotlin test dependencies under `/* Kotlin */` or under `/* Testing */`?